src/components/OmouComponents/IdleLogout.js

```diff
@@ -82,12 +90,16 @@ const IdleLogout = () => {
     // Time(ms) user has to click that they're still here before they're logged out
     const modalTimeout = 180000;
 
+    // State variable being updated by the idle timer - when this variable reaches 0 the
+    // modal will display
     const [remainingMsUntilPrompt, setRemainingMsUntilPrompt] = useState(idleTimeout);
```

Comment:
Explain what is an idle timer - i'm assuming this is the timer that's tracking the time after the last time a user has performed something